src/Generator/Passes/CleanCommentsPass.cs

```diff
@@ -14,7 +14,7 @@ public bool VisitBlockCommand(BlockCommandComment comment)
         {
             return true;
         }
-
+        bool refFlag = false;
```

Comment:
You have accidentally committed leftover code from the comments, remove all of it.